sources/packages/backend/apps/api/src/services/education-program/education-program.service.ts

```diff
@@ -331,6 +331,33 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
       });
       queryParams.push(`%${paginationOptions.searchCriteria}%`);
     }
+    if (paginationOptions.programNameSearch) {
```

Comment:
Please correct me if i am wrong, the paginationOptions, if searched with only one criteria, paginationOptions.searchCriteria is used else the paginationOptions.programNameSearch and paginationOptions.locationNameSearch is pushed. If that is the case, either paginationOptions.searchCriteria should be pushed or the  paginationOptions.programNameSearch and paginationOptions.locationNameSearch.

Reply:
`searchCriteria` should not be used for programs.
@sh16011993 what if we change the service models as below to ensure the `searchCriteria` will not be present.
```ts
export interface BasePaginationOptions {
  sortField?: string;
  sortOrder?: FieldSortOrder;
  page: number;
  pageLimit: number;
}

/**
 *  Pagination option.
 */
export interface PaginationOptions extends BasePaginationOptions {
  searchCriteria?: string;
}

export interface ProgramPaginationOptions extends BasePaginationOptions {
  programNameSearch?: string;
  locationNameSearch?: string;
  status?: string;
}
```